docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Accessing the Cloud with Viam's Client SDKs"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "robot api", "cloud management"]
+---
+
+The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of through the [Viam app](https://app.viam.com/).
+Functionality includes:
+
+- create and manage organizations, locations, and individual robots
+- manage permissions and authorization
+- create and manage fragments
+
+{{% alert title="Support Notice" color="note" %}}
+
+As of August 24, 2023, the cloud app API is only available in Python.
+
+{{% /alert %}}
+
+## API
+
+{{% alert title="Tip" color="tip" %}}
```

Comment:
Yes! That looks like just what we need here!!

Reply:
I'd done it as a tip initially because it is different so didn't want it to look as though it wasn't different, but I think it'll work! will do

Reply:
Oh also I may have looked and seen this as a tip: https://docs.viam.com/services/vision/#api

Reply:
Ooh I could see that tip having a similar problem there as well: skippable! Anyways, definitely out of scope for this PR!

Reply:
Indeed, and on many other service pages as well! Though personally I think the tips are so colorful and eye-catching that it might be ok esp if we changed them to important or something. But indeed a problem for another PR!